dias/analyzers/sensitivity_analyzer.py

```diff
@@ -0,0 +1,273 @@
+        
+            t0 = time.time()
+            
+            # Look up inputmap
+            inputmap   = tools.get_correlator_inputs(ephemeris.unix_to_datetime(timestamp0),
+                                                   correlator=self.correlator)
+            # Load index map and reverse map
+            data       = andata.CorrData.from_acq_h5(files, 
+                                               datasets=['reverse_map', 'flags/inputs'],
+                                               apply_gain=False, renormalize=False)
+            
+            # Determine axes
+            nfreq     = data.nfreq
+            nblock    = int(np.ceil(nfreq / float(self.nfreq_per_block)))
+            
+            timestamp = data.time
+            ntime     = timestamp.size
+
+            # Get baselines
+            prod, prodmap, dist, conj, cyl, scale = self.get_baselines(data.index_map, inputmap)
+
+            # Determine groups
+            polstr = np.array(sorted(prod.keys()))
+            npol   = polstr.size
+            
+            t0 = time.time()
+            
+            # Calculate counts
+            cnt = np.zeros((data.index_map['stack'].size, ntime), dtype=np.float32)
+            
+            if np.any(data.flags['inputs'][:]):
+                for pp, ss in zip(data.index_map['prod'][:], data.reverse_map['stack']['stack'][:]):
+                    cnt[ss, :] += (data.flags['inputs'][pp[0], :] * data.flags['inputs'][pp[1], :])
+            else:
+                for ss, val in Counter(data.reverse_map['stack']['stack'][:]).iteritems():
+                    cnt[ss, :] = val
+            
+            self.logger.info("Took %0.1f seconds to get the counts" % (time.time() - t0))
+
+            # Initialize arrays
+            var 	= np.zeros((nfreq, npol, ntime), dtype=np.float32)
+            counter = np.zeros((nfreq, npol, ntime), dtype=np.float32)
+
+            # Loop over frequency blocks
+            for block_number in range(nblock):
+        
+                t0 = time.time()
+        
+                fstart   = block_number * self.nfreq_per_block
+                fstop    = min((block_number + 1) * self.nfreq_per_block, nfreq)
+                freq_sel = slice(fstart, fstop)
+        
+                self.logger.info("Processing block %d (of %d):  %d - %d" % (block_number+1, nblock, fstart, fstop))
+
+                bdata    = andata.CorrData.from_acq_h5(files, freq_sel=freq_sel,
+                                                    datasets=['flags/vis_weight'],
+                                                    apply_gain=False, renormalize=False)
+
+                t0       = time.time()
+                bflag    = (bdata.weight[:] > 0.0).astype(np.float32)
+                bvar     = tools.invert_no_zero(bdata.weight[:])
+                
+                # Loop over polarizations
+                for ii, pol in enumerate(polstr):
+
+                    self.logger.info("Processing Pol %s" % pol)
```

Comment:
```suggestion
                    self.logger.debug("Processing Pol %s" % pol)
```

Reply:
To reduce the logging info, I plan to remove the pol logging information. Logging would only add information about processing of each frequency block.